statsmodels/genmod/cov_struct.py

```diff
@@ -938,6 +951,14 @@ class Equivalence(CovStruct):
     using this covariance structure, missing data should be removed
     before constructing the model.
 
+    If using `labels`, after a model is defined using the covariance
+    structure it is possible to remove a label pair from the second
+    level of the `pairs` dictionary to force the corresponding
+    covariance to be zero.
+
+    The `pairs` dictionary is modified by the function so the user
+    should pass a copy if it is needed elsewhere.
```

Comment:
I need to think about this, in general we avoid changing original arrays in functions.